apache-rat-core/src/main/java/org/apache/rat/Report.java

```diff
@@ -130,9 +136,15 @@ public String desc() {
     /**
      * Adds license headers to files missing headers.
      */
-    private static final OptionGroup ADD = new OptionGroup()
-            .addOption(new Option("a", false,
-                    "(deprecated) Add the default license header to any file with an unknown license.  Use '-A' or ---addLicense instead."))
+    // TODO rework when Commons-CLI version 1.7.1 or higher is available.
```

Comment:
Should we add a new jira task for these refactorings in case 1.7.1 is released? So that they are not forgotten? We could even add this ticket to the TODOs

Reply:
I have created RAT-380 for that

Reply:
So we can merge this PR and open a new one for the refactorings :)